src/pages/models/[id].tsx

```diff
@@ -30,21 +30,85 @@ interface Props {
     modelData: Model;
 }
 
-const renderTags = (tags: string[]) => (
+const extraModelProperties = [
+    { key: 'trainingIterations', type: 'number' },
+    { key: 'trainingEpochs', type: 'number' },
+    { key: 'trainingBatchSize', type: 'number' },
+    { key: 'trainingHRSize', type: 'number' },
+    { key: 'trainingOTF', type: 'string' },
+    { key: 'dataset', type: 'string' },
+    { key: 'datasetSize', type: 'number' },
+    { key: 'pretrainedModelG', type: 'string' },
+    { key: 'pretrainedModelD', type: 'string' },
```

Comment:
Those are model ids. But we probably need another input for them, so maybe fixing this should be its own PR?

Reply:
I hide these right now anyway

Reply:
Oh, that's fine then.